README.md

```diff
@@ -1,44 +1,52 @@
+<p align="center">
+    <img alt="Plone Logo" width="200px" src="https://raw.githubusercontent.com/plone/.github/main/plone-logo.png">
+</p>
+
+<h1 align="center">
+  Cookieplone Templates
+</h1>
+
+<div align="center">
+
 ![GitHub](https://img.shields.io/github/license/plone/cookiecutter-plone)
 [![Black code style](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
 
-# Cookieplone Templates
+</div>
 
-Powered by [Cookieplone](https://github.com/plone/cookieplone), this is a collection of templates intended to be used by Plone integrators.
+## About
 
-## Getting Started 🏁
+Powered by [Cookieplone](https://github.com/plone/cookieplone "Link to the GitHub repository of Cookieplone" ), a collection of templates intended to be used by Plone integrators.
```

Comment:
Make this a complete sentence, instead of a fragment.

```suggestion
Powered by [Cookieplone](https://github.com/plone/cookieplone "Link to the GitHub repository of Cookieplone"), this project is a collection of templates intended to be used by Plone integrators.
```